cmd/config.go

```diff
@@ -35,6 +35,11 @@ func setGlobalConfigPath(configPath string) {
 	customConfigPath = configPath
 }
 
+// Sets the global config path to empty string.
+func resetGlobalConfigPath() {
+	setGlobalConfigPath("")
+}
+
```

Comment:
Is this function expected to be used in normal functioning of the serve? If not move it to test-utils_test.go.